### ISSUE TITLE
Error for batch size of 1

### ISSUE THREAD

**Issue by friggog** [zhixuanli/StereoNet#2]
https://github.com/zhixuanli/StereoNet/blob/44cf5a852e578b2d769af40d86298847c481279f/models/stereonet.py#L215

You need to specify `dim=1` on this line or all dimensions are squeezed when batch size is 1

**Comment by zhixuanli** [comment]
Yes, you are right! I'll fix it soon, for it is Chinese lunar new year now😉.
I'll deal with it ASAP when I get back to school.
